cmd/gh-classroom/clone/student-repos/student-repos.go

```diff
@@ -98,6 +103,7 @@ func NewCmdStudentRepo(f *cmdutil.Factory) *cobra.Command {
 	cmd.Flags().StringVarP(&directory, "directory", "d", ".", "Directory to clone into")
 	cmd.Flags().IntVar(&page, "page", 1, "Page number")
 	cmd.Flags().IntVar(&perPage, "per-page", 30, "Number of accepted assignments per page")
```

Comment:
>  I found that 20 seems to be a low enough number to allow the download to complete (though it should be noted that I still got a 500 twice trying to download 4 pages).

Since 20 was still too high, we should probably lower it just to be safe.

See https://github.com/github/gh-classroom/issues/38#issue-1899642573
```suggestion
	cmd.Flags().IntVar(&perPage, "per-page", 15, "Number of accepted assignments per page")
```

Reply:
@zrdaley Awesome thanks for the feedback